src/DIRAC/WorkloadManagementSystem/Agent/PilotSyncAgent.py

```diff
@@ -40,13 +40,17 @@ def __init__(self, *args, **kwargs):
     # '/opt/dirac/webRoot/www/pilot'
     self.saveDir = ''
     self.uploadLocations = []
+    self.includeMasterCS = True
 
   def initialize(self):
     """ Initial settings
     """
     self.workingDirectory = self.am_getOption('WorkDirectory')
     self.saveDir = self.am_getOption('SaveDirectory', self.saveDir)
     self.uploadLocations = self.am_getOption('UploadLocations', self.uploadLocations)
+    includeMasterCS = self.am_getOption('IncludeMasterCS', self.includeMasterCS)
+    if isinstance(includeMasterCS, six.string_types) and includeMasterCS.lower() in ['n', 'no', 'false']:
```

Comment:
Can the includeMasterCS be a string at this point ?

Reply:
This check might be redundant, I am not sure tbh.